app/views/users/show.html.erb

```diff
@@ -11,7 +21,6 @@
   </div>
   <div>
     <h2>Bio</h2>
-
     <% @user.posts.each do |post| %>
       <p><%= post.body %></p>
     <% end %>
```

Comment:
- [ ] It happens that there's an issue with the code in your here. The error is saying that there's an undefined method body for the post object.

In your Post model, it seems like you don't have a body attribute defined. You might want to replace post.body with post.text or whatever attribute you're using to store the content of the post.

So, the corrected code might look like this:

```
<div>
  <h2>Bio</h2>
  <% @user.posts.each do |post| %>
    <p><%= post.text %></p>
  <% end %>
</div>

```